jsbits/delegate.js

```diff
@@ -1,3 +1,18 @@
+var registeredCallbacks = [];
+
+/* Callbacks in ghcjs need to be released. With this function one can register
+   callbacks that should be released right before diffing.
+*/
+function registerCallback(cb) {
+    registeredCallbacks.push(cb);
+}
+
+/* This clears the callbacks. */
+function clearCallbacks() {
+    for (var i in registeredCallbacks)
+      h$release(registeredCallbacks[i]);
```

Comment:
@FPtje shouldn't `registeredCallbacks` be reset to `[]` here?

Reply:
Good point